packages/api-docs-builder-core/baseUi/projectSettings.ts

```diff
@@ -43,4 +43,6 @@ export const projectSettings: ProjectSettings = {
   translationPagesDirectory: 'docs/translations/api-docs-base',
   generateClassName: generateUtilityClass,
   isGlobalClassName: isGlobalState,
+  // #major-version-switch
```

Comment:
@mnajdova is `#major-version-switch` the correct flag? I don't know if we have another flag for it.

Reply:
#default-branch-switch is much more used, honestly first time I see the #major-version-switch :)

Reply:
Come from herer. I will harmonize it :+1: